src/open_inwoner/utils/validators.py

```diff
@@ -94,3 +94,20 @@ def __call__(self, image_id):
 
     def __eq__(self, other):
         return self.width == other.width and self.height == other.height
+
+
+class DiversityValidator:
+    def validate(self, password, user=None):
+        if password.isalpha() or password.isupper() or password.islower():
```

Comment:
This check is not correct and doesn't catch single case and digits (eg `"password123"`)

Reply:
`"password123".islower()` returns `True`. It's not obvious from the name of the method, but Python is smart enough to check only the letters.

Reply:
I'm sorry, I'm an idiot and didn't pay attention. 

I added `password123` to the test case and it failed, but it wasn't because this validator (but because the generic validator).